README.md

```diff
@@ -6,35 +6,43 @@ This repository applies [BERT](https://github.com/google-research/bert) to named
 
 ## Installation
 
-(will make available via `pip install` and `conda install` when project is closer to completion - hit the Watch button if you'd like to be updated on progress!)
+**NOTE**: Since there are quite a few requirements, creating a virtual environment for this repo is strongly recommended.
+
+**Instructions**: Use `conda` to create the environment. Currently environment creation via pip is not supported.
+
+To create the environment with `conda`, all you need to do is navigate to the `Multilingual_NER` directory and run the following in the command line:
+
+`conda env create -f environment.yml`
```

Comment:
great addition. This works well. However, when I run the streamlit demo, it is missing the xx_ent_wiki_sm, so python -m spacy download xx_ent_wiki_sm should go somewhere in your install instructions (inside the environment)